src/main/java/uk/gov/hmcts/reform/prl/mapper/citizen/CaseDataApplicantElementsMapper.java

```diff
@@ -200,12 +199,6 @@ private static List<Element<ChildrenAndApplicantRelation>> buildChildAndApplican
     }
 
     private static void checkAndupdatePcqIdForParty(PartyDetails party, String applicantPcqId) {
-        if (ObjectUtils.isNotEmpty(party.getUser())) {
-            if (StringUtils.isEmpty(party.getUser().getPcqId())) {
-                party.getUser().setPcqId(applicantPcqId);
-            }
-        } else {
-            party.setUser(User.builder().pcqId(applicantPcqId).build());
-        }
+        party.setUser(User.builder().pcqId(applicantPcqId).build());
```

Comment:
Explain this code change please?

Reply:
This code is required for sonar code coverage and has already been discussed with Sairam.